.travis.yml

```diff
@@ -7,11 +7,10 @@ script: bundle exec rake travis
 matrix:
   fast_finish: true
   include:
-    - rvm: 1.9.3
-    - rvm: 2.0.0
-    - rvm: 2.1.0
-    - rvm: 2.1.5
-    - rvm: 2.1.5
+    - rvm: 2.0
+    - rvm: 2.1
+    - rvm: 2.2
+    - rvm: 2.2
```

Comment:
I'm not entirely sure. It appears that master has duplicate entries as well. https://github.com/fog/fog/blob/master/.travis.yml#L22-L25

This is how the Travis docs recommend, but we end up with several more runs (each version against edge), which could be good:

```yml
rvm:
  - 2.0
  - 2.1
  - 2.2
  - jruby-19mode
  - jruby-head

gemfile:
  - Gemfile
  - gemfiles/Gemfile-edge

matrix:
  fast_finish: true

  allow_failures:
    - rvm: jruby-head
```